src/static/js/modules/prepare-worksheets/inputs/button-grading-group.js

```diff
@@ -11,42 +11,39 @@ function ButtonGradingGroup(options) {
   // Note bind()'s lack of IE8 support.
   var _self = this;
 
-  var _grades = require('./input-graded-grades');
-
-  // The grade of this input.
-  var grade = _grades.UNSET;
-  var lastNode;
-
+  this.row = options.row;
```

Comment:
Is `row` used outside of this module? e.g. is something like this code somewhere else in the codebase:

```javascript
var ButtonGradingGroup = require( 'button-grading-group' );
...
var group = ButtonGradingGroup.create( options );
console.log( group.row ); // <-- Is this happening somewhere else where there's a need to get access to the row externally to the module?
```